staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -343,7 +343,7 @@ func NewConfig(codecs serializer.CodecFactory) *Config {
 		}
 
 		hash := sha256.Sum256([]byte(hostname))
-		id = "kube-apiserver-" + strings.ToLower(base32.StdEncoding.WithPadding(base32.NoPadding).EncodeToString(hash[:16]))
+		id = "apiserver-" + strings.ToLower(base32.StdEncoding.WithPadding(base32.NoPadding).EncodeToString(hash[:16]))
```

Comment:
To support distinct, unrelated aggregated API servers running on the same host we need to include the identity label value in this ID.  Thus I think for KAS this value is correct (basically if you change this here you need to override it in `buildGenericConfig` to the current value).

Reply:
It seems easier to use the identity value for the hash and keep the `apiserver-` prefix consistent.

Reply:
Whatever we decide, the docs for the label should include a brief mention of that detail.

Reply:
> It seems easier to use the identity value for the hash and keep the `apiserver-` prefix consistent.

If you mean something like `hash := sha256.Sum256([]byte(hostname + labelValue))` (but implemented using the cryptobyte API) then yeah that would work as well.